backend/services/container_migration_service.py

```
@@ -0,0 +1,260 @@
+
+            # Create volume mapping and transfer data
+            volume_mapping = {}
+            snapshots = []
+
+            for volume in volumes:
+                target_path = f"{request.target_base_path}/{volume.source.split('/')[-1]}"
+                volume_mapping[volume.source] = target_path
+
+                if use_zfs:
+                    # ZFS migration
+                    dataset_name = await self.zfs_ops.get_dataset_name(volume.source)
```

Ruff (0.11.9)

Local variable `dataset_name` is assigned to but never used

Remove assignment to unused variable `dataset_name`

(F841)